docs/blog/2020-08-10-drand-launches-v1.0.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Drand launches v1.0, and the League of Entropy becomes a production service!
+summary: Welcome to the brand new website for the drand project! Drand is now in production, providing publicly verifiable randomness as a foundational internet service.
+date: 2020-08-10
+tags:
+    - Release
+    - Major
+    - League of Entropy
+---
+
+![Placeholder launch image](/images/social-card.png)
```

Comment:
^ labelled as a placeholder. It seems a little redundant to have a drand logo as a cover image in the drand website. It seems we have the opengraph image set so the social card will be used for SNS previews anyway.

Reply:
It is a placeholder until this issue is resolved: https://github.com/protocol/design-shop/issues/344

Reply:
Missed that one, thanks!